	<launch>
		<node name="dvl_raw" pkg="data_pub" exec="dvl_raw" output="screen" />
		<node name="dvl_odom" pkg="data_pub" exec="dvl_odom" output="screen" />
		<node name="peripheral" pkg="data_pub" exec="peripheral" output="screen" />
	</launch>